src/components/map/GoogleMap.svelte

```diff
@@ -66,21 +67,65 @@
 	}
 
 	function addMarker(s) {
-		const glyphImg = document.createElement('img');
-		glyphImg.src = busIcon;
+		const markerElement = document.createElement('div');
```

Comment:
Do you think you can turn the non-Google custom marker code into a new, reusable client-side component? https://svelte.dev/docs/client-side-component-api.

Reply:
Could you please provide more details on how you envision turning the custom marker code into a reusable client-side component?